src/calculateRentalCost.js

```diff
@@ -5,6 +5,20 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const baseRateOneDay = 40;
+  const THREE_DAYS = 3;
+  const SEVEN_DAYS = 7;
+  const DISCOUNT_20 = 20;
+  const DISCOUNT_50 = 50;
+  let totalCost = days * baseRateOneDay;
```

Comment:
Consider using `const` here instead of `let` for `totalCost` since it's not being reassigned but only modified. This can help prevent unintentional reassignments.